bin/create_k8s_token.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/sh
+
+##
+# little script that will create a vault token for the secret puller, and then store it as a 
```

Comment:
The intention is that it will run on some admin box that has access to both
vault and to the k8s cluster.

On Fri, Jul 8, 2016 at 1:16 PM, Michael Grosser <notifications@github.com>
wrote:

> In bin/create_k8s_token.sh
> <https://github.com/zendesk/samson_secret_puller/pull/14#discussion_r70134491>
> :
>
> > @@ -0,0 +1,122 @@
> > +#!/bin/sh
> > +
> > +##
> > +# little script that will create a vault token for the secret puller, and then store it as a
>
> this will run locally or on a pod ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson_secret_puller/pull/14/files/aee139efc293a35b8a14878d4584bce9defd4787#r70134491>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAJfohG3J6fysEb17KeMWLFSQcyMmcWPks5qTrAsgaJpZM4JIXEf>
> .
>